server/boot.js

```diff
@@ -0,0 +1,20 @@
+import fs from 'fs';
+import path from 'path';
+import mkdirp from 'mkdirp';
+import Conf from './context/conf';
+import Router from './router';
+import Instrumenter from './services/instrumenter';
+
+export default Boot = {
+  startup() {
+    // Create reports output folder if not exists
+    let outputFolder = path.resolve(Conf.COVERAGE_APP_FOLDER, `.${path.sep}${Conf.COVERAGE_EXPORT_FOLDER}`);
```

Comment:
`path.join(Conf.COVERAGE_APP_FOLDER, Conf.COVERAGE_EXPORT_FOLDER)` can make the job